Sample Data/ASIM/Infoblox_BloxOne_ASimDns_IngestedLogs.csv

```diff
@@ -0,0 +1,21 @@
+asdfvasd-3a80-4066-adf8-0xxxxxxx,"6/25/2024, 3:42:19 PM",Infoblox,Data Connector,2.1.3,DNS Response,DNS Response IN A NOERROR,1,,"InfobloxDNSView=;InfobloxDNSQClass=IN;InfobloxDNSQType=A;InfobloxDNSQFlags=+EV;InfobloxDNSRCode=NOERROR;InfobloxAnCount=1;InfobloxNsCount=0;InfobloxArCount=0;InfobloxB1Region=us-west-1;InfobloxB1ConnectionType=remote_office;InfobloxB1OPHName=;InfobloxB1OPHIPAddress=;InfobloxB1Network=;InfobloxB1SrcOSVersion=;InfobloxB1DHCPFingerprint=;InfobloxB1DNSTags=APP_Uncategorized,CAT_Technology - Other",,DNS,,www.example.com.,,,,,,,,,,,,,,,,,,,,,,,,,1.1.1.1,2.2.2.2,,,,,,,,,,,,,,,,"""www.example.com. 291 IN A 93.184.215.14""",,,,,,,,,,,,TCP,,,,,,,,,,,,,,,,,,21388,3.3.3.3,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,ZTP_MSSentinel1604ForOva_92033666899467253,OpsManager,,CEFDataConnector1406,CommonSecurityLog,/subscriptions/sub_id/resourcegroups/resourcegrpname_1/providers/microsoft.compute/virtualmachines/cefdataconnector1406
```

Comment:
what does Message field contain? Can you check if this field contain SrcPortNumber, DNSResponseName etc.?
![image](https://github.com/user-attachments/assets/0ac0f1c1-ee91-4a3b-8142-12cd640c284e)

Reply:
Please extract this field and map with relevant fields post extraction.

Reply:
No, This field doesn't contain SrcPortNumber, DNSResponseName etc.